### PR TITLE
Clear the shares after the test like storages and files

### PR DESCRIPTION
**Pull request by nickvergessen**
Should fix failing master tests

@PVince81 @DeepDiver1975 @MorrisJobke

### CODE REVIEW

File: tests/lib/testcase.php [hunk 1]
```diff
@@ -140,25 +143,33 @@ public static function tearDownAfterClass() {
 	}
 
 	/**
+	 * Remove all entries from the share table
+	 *
+	 * @param IQueryBuilder $queryBuilder
+	 */
+	static protected function tearDownAfterClassCleanShares(IQueryBuilder $queryBuilder) {
+		$queryBuilder->delete('share')
+			->execute();
```
Comment (MorrisJobke) [inline tests/lib/testcase.php]: Why not check here if there was something deleted and throw an exception like "clean up your mess"?

**Comment by PVince81** [comment]
:+1:

File: tests/lib/testcase.php (hunk 1 above)
Comment (DeepDiver1975) [inline tests/lib/testcase.php]: valis approach for 9.0 - for 8.2 we keep it like this

Comment (MorrisJobke) [inline tests/lib/testcase.php]: Fine for now.

**Comment by MorrisJobke** [comment]
:+1: from my side.

**Comment by karlitschek** [comment]
please backport :+1:

**Comment by MorrisJobke** [comment]
> please backport :+1:

I will do

**Pull request merged**